utils/evm/external.ts

```
@@ -1,73 +1,180 @@
+        `[TxFrame] Sending transaction to ${to} with value ${value} on chain ${
+          chain.name || chain.id
+        }`
+      );
+
+      const transaction = prepareTransaction({
+        to,
+        value: value ? BigInt(value) : undefined,
+        chain,
+        client: thirdwebClient,
+        data:
+          data && data.startsWith("0x") ? (data as `0x${string}`) : undefined,
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)